std/experimental/color/rgb.d

```diff
@@ -0,0 +1,446 @@
+// Written in the D programming language.
+
+/**
+    This module implements the RGB _color type.
+
+    Authors:    Manu Evans
+    Copyright:  Copyright (c) 2015, Manu Evans.
+    License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+    Source:     $(PHOBOSSRC std/experimental/color/rgb.d)
+*/
+module std.experimental.color.rgb;
+
+import std.experimental.color;
+import std.experimental.color.conv;
+
+import std.traits: isInstanceOf, isNumeric, isIntegral, isFloatingPoint, isSigned, isSomeChar, Unqual;
+import std.typetuple: TypeTuple;
+import std.typecons: tuple;
+
+@safe: pure: nothrow: @nogc:
+
+enum isValidComponentType(T) = isIntegral!T || isFloatingPoint!T;
+
+
+/**
+Detect whether $(D T) is an RGB color.
+*/
+enum isRGB(T) = isInstanceOf!(RGB, T);
+
+
+// DEBATE: which should it be?
```

Comment:
as you did post it in the review thread and in the PR comments and remove the DEBATE line